README.md

```diff
@@ -21,6 +21,14 @@ Möbius is a Smart Contract that runs on Ethereum that offers trustless autonomo
 
 :point_right: **See the full [Tutorial](./mobiusTutorial.md)**
 
+-----------------------------------------------------
+
+Möbius supports ether transactions and ERC20 compatible token transactions.
+In order to use the Mixer with ERC20 compatible tokens, one has to use the `DepositERC20Compatible` and `WithdrawERC20Compatible` functions.
```

Comment:
I would use "`DepositERC20Compatible` and `WithdrawERC20Compatible` functions must be used"